### PR TITLE
Adding Session Id support in Publish to service bus task

### PR DESCRIPTION
**Pull request by rohit-batra**

### CODE REVIEW

File: Tasks/PublishToAzureServiceBusV1/task.json [hunk 1]
```diff
@@ -52,14 +52,23 @@
                 "editorExtension": "ms.vss-services-azure.azure-servicebus-message-grid"
             }
         },
+		{
+			"name": "sessionId",
+			"type": "string",
+			"label": "Session Id for Message",
```
Comment (bansalaseem) [inline Tasks/PublishToAzureServiceBusV1/task.json]: Rename this to Session Id

File: Tasks/PublishToAzureServiceBusV1/task.json [hunk 2]
```diff
@@ -52,14 +52,23 @@
...
+			"required": false,
+			"defaultValue": "",
+			"helpMarkDown": "This is the session id whith which you want to post the message. In case not given , default Guid will be assigned to it. Used for Session Based Queues, for Non Session Based Queues, it would not matter.",
```
Comment (bansalaseem) [inline Tasks/PublishToAzureServiceBusV1/task.json]: Please fix up the help markdown as we are no longer generating default Guid

File: Tasks/PublishToAzureServiceBusV1/task.json (hunk 1 above)
Comment (rohit-batra) [reply Tasks/PublishToAzureServiceBusV1/task.json]: DOne

File: Tasks/PublishToAzureServiceBusV1/task.json (hunk 2 above)
Comment (rohit-batra) [reply Tasks/PublishToAzureServiceBusV1/task.json]: Done

File: Tasks/PublishToAzureServiceBusV1/Strings/resources.resjson/en-US/resources.resjson [hunk 3]
```diff
@@ -3,11 +3,13 @@
...
   "loc.description": "Sends a message to azure service bus using a service connection (no agent required).",
   "loc.instanceNameFormat": "Publish to Azure Service Bus",
-  "loc.group.displayName.signingDetails": "Signing Properties",
+  "loc.group.displayName.advancedProperties": "Advanced",
   "loc.input.label.connectedServiceName": "Azure Service Bus service connection",
   "loc.input.help.connectedServiceName": "Select an Azure Service Bus service connection.",
   "loc.input.label.messageBody": "Message body",
   "loc.input.help.messageBody": "Enter the json messageBody.",
+  "loc.input.label.sessionId": "Session Id",
+  "loc.input.help.sessionId": "This is the session id with which you want to post the message. If not given for Session Based Queues, Publishing of message will fail for such Queueus. For Non Session Based Queues, it will not matter.",
```
Comment (bansalaseem) [inline Tasks/PublishToAzureServiceBusV1/Strings/resources.resjson/en-US/resources.resjson]: There are spelling mistakes "Queueus", incorrect case "Session Based Queues, Publishing", "for such Queues", "Non Session Based Queues" and its directed towards user with words like you..

Change the complete sentence to "Session id with which message is published. For session based queues, publishing fails if value not specified and for non session based queues, value is ignored."

File: Tasks/PublishToAzureServiceBusV1/task.json [hunk 4]
```diff
@@ -103,6 +112,7 @@
                 "connectionString": "$(endpoint.serviceBusConnectionString)",
                 "serviceBusQueueName": "$(endpoint.serviceBusQueueName)",
                 "messageBody": "$(messageBody)",
+				"sessionId": "$(sessionId)",
```
Comment (bansalaseem) [inline Tasks/PublishToAzureServiceBusV1/task.json]: Fix the indentation

File: Tasks/PublishToAzureServiceBusV1/Strings/resources.resjson/en-US/resources.resjson (hunk 3 above)
Comment (rohit-batra) [reply Tasks/PublishToAzureServiceBusV1/Strings/resources.resjson/en-US/resources.resjson]: This sentence is factually incorrect , so will not change it for now . Will discuss with you.

Comment (rohit-batra) [reply Tasks/PublishToAzureServiceBusV1/Strings/resources.resjson/en-US/resources.resjson]: Also i dont think word "you" should not be used as i can see multiple such usages in task markdown. But taking the change to move forward.

**Pull request merged**